.circleci/config.yml

```diff
@@ -69,41 +71,26 @@ workflows:
           requires:
             - build
       - docs-validation
-      - ginkgo
       - git-validation
-      - integration:
-          requires:
-            - build
-            - build-test-binaries
+      - integration
```

Comment:
Is this line valid?

Reply:
Yes :) there are no parameters needed any more